Sources/Vapor/Utilities/URI.swift

```diff
@@ -1,6 +1,52 @@
 import CURLParser
 
 public struct URI: ExpressibleByStringInterpolation, CustomStringConvertible {
+    /// A URI's scheme.
+    public enum Scheme: ExpressibleByStringInterpolation {
+        /// HTTP
+        case http
+        
+        /// HTTPS
+        case https
+        
+        /// HTTP over Unix Domain Socket Paths. The socket path should be encoded as the host in the URI, making sure to encode any special characters:
+        /// ```
+        /// host.addingPercentEncoding(withAllowedCharacters: .urlHostAllowed)
+        /// ```
+        /// Do note that URI's initializer will encode the host in this way.
+        case httpUDS
```

Comment:
```suggestion
        case httpUnixDomainSocket
```
Swift style recommends against abbreviation. This will be a long one, but luckily autocomplete helps us here haha.

Reply:
I had it as the long one originally, and waffled a few too many times. I'll change it back 😉